### DISCUSSION TITLE
Use SEB with SIP disabled

### DISCUSSION THREAD

**Discussion by StadeRochelais17** [SafeExamBrowser/seb-mac#361] [General]
Hi,

is it possible, is there a workaround ,to use SEB with SIP disabled ?
When I launch it, SEB is locked, I have the message "SIP is disabled, enable SIP and restart SEB"

In my school, we have disable SIP on all Apple machines, because of rights problem software deployment.
We use Munki to deploy apps, and with SIP enable and given full access disk to Munki, we have some rights problem for few software ...
So we must disable SIP

Regards

**Comment by danschlet** [comment]
No, we won't support turning off SIP, because it is an essential security feature of macOS and Apple doesn't encourage using macOS without it. Also it is an essential feature for ensuring SEB security.

You should reach out to Munki and make sure that their software works correctly with SIP active and all third party software.
